### PR TITLE
Aariste maxpararellism bacpac

### PR DESCRIPTION
**Pull request by Splaxi**

### CODE REVIEW

**Comment by Splaxi** [comment]
You also have a few files that shouldn't be part of your branch: 
```
.vs/VSWorkspaceState.json
.vs/d365fo.tools/v16/.suo
.vs/slnx.sqlite
```

File: d365fo.tools/internal/functions/invoke-sqlpackage.ps1 [hunk 1]
```diff
@@ -159,6 +164,11 @@ function Invoke-SqlPackage {
         $null = $Params.Add("/ModelFilePath:`"$ModelFile`"")
     }
 
+    if (-not [system.string]::IsNullOrEmpty($MaxParallelism)) {
+    {
```
Comment (Splaxi) [inline d365fo.tools/internal/functions/invoke-sqlpackage.ps1:168]: This one is just 1 to many...

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Splaxi) [inline .vs/VSWorkspaceState.json:1]: This file should be deleted.

**Comment by aariste** [comment]
I've just seen the extra {.

The 3 files are being added by Visual Studio each time I edit any of the other files. I'm using GitHub extension for VS, I'll download GitHub desktop and use it instead.

**Comment by Splaxi** [comment]
Perfect! 👍

**Comment by Splaxi** [comment]
I can fix the whitespace, if you just fix the files that shouldn't be there.

A simple way to fix it, is to just copy the 3 files when they are perfect and the create a new branch via Github Desktop and edit the files from outside VS...

**Comment by aariste** [comment]
What if I add the .vs folder to .gitignore? Would that solve it too?

I'm sorry Mötz, I think I've created more problems than what I though would solve (which wasn't even a problem...)

**Comment by Splaxi** [comment]
If you have the 3 files somewhere on your machine, I can walk you through how to do it right, using Github Desktop.

**Comment by aariste** [comment]
I've got them located, they're in the solution folder.

**Pull request closed without merging**